packages/worker/src/worker.ts

```diff
@@ -1334,7 +1344,7 @@ export class Worker {
       // The only way for this observable to be closed is by state changing to DRAINED meaning that all in-flight activities have been resolved and thus there should not be any heartbeats to send.
       this.takeUntilState('DRAINED'),
       tap({
-        complete: () => this.log.debug('Heartbeats complete'),
+        complete: () => this.logger.debug('Heartbeats complete'),
       }),
```

Comment:
Do you need logAttributes here?

Reply:
This is an Activity Worker lifecycle event, not specific to a single activity. Hence, no pertinent `logAttributes` to report.